doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -452,3 +452,21 @@ of ``/sys/block/<drive>/queue/optimal_io_size``.
 .. confval:: bluestore_min_alloc_size_ssd
 .. confval:: bluestore_use_optimal_io_size_for_min_alloc_size
 
+DSA (Data Streaming Accelerator Usage)
+======================================
+
+If you want to use the DML library to drive DSA device for offloading
+read/write operations on Persist memory in Bluestore. You need to install
+DML and idxd-config library in your machine with SPR (Sapphire Rapids) CPU.
+Refer following site for more details.
+
+.. __: https://github.com/intel/DML
+.. __: https://github.com/intel/idxd-config
```

Comment:
@optimistyzy  these hyperlinks lead to following warning:

```
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/rados/configuration/bluestore-config-ref.rst:: ERROR: Anonymous hyperlink mismatch: 1 references but 3 targets.
```

and the generated HTML does not look right. see https://ceph--44230.org.readthedocs.build/en/44230/rados/configuration/bluestore-config-ref/#dsa-data-streaming-accelerator-usage

could you please use the following notation instead? something like
```rst
- `DML <https://github.com/intel/DML>`__
- `libaccel-config <https://github.com/intel/idxd-config/>`__
```

Reply:
ack